package.json

```diff
@@ -205,19 +205,17 @@
     "electron-builder": "^22.14.5",
     "electron-devtools-installer": "^3.2.0",
     "electron-rebuild": "^3.2.2",
-    "eslint": "^8.4.1",
-    "eslint-config-airbnb-typescript": "^6.3.1",
-    "eslint-config-erb": "^0.3.0",
+    "eslint": "^7.32.0",
```

Comment:
Does this bring any other security alerts with the downgrade?

Reply:
There are still some sec vulnerabilities but nothing critical

Reply:
nevermind - I take that back - give me a few mins

Reply:
No worries, suggest just running `npm audit fix` or just do `npm update --save` to update all deps and see if we still have alerts and the demo runs fine.

Reply:
Okay now there isnt!